### DISCUSSION TITLE
About Annunciation Transfer (depending on the version it should not be moved)

### DISCUSSION THREAD

**Discussion by d4h0y0sr** [DivinumOfficium/divinum-officium#4208] [General]
Hi, i was reviewing the behavior of moving the annunciation, and it seems wrong.
According to my missal 1962, this feast only moves when it falls on easter week, could you help me with more information about it?

**Comment by APMarcello3** [comment]
See X. De Translatione Festorum for pre-1960 rubrics and 96a of 1960 Codex Rubricarum.
